src/index.js

```diff
@@ -26,11 +26,6 @@ const listItems = [
     description: "Join morning session call",
     completed: false,
     index: 4
-  },
```

Comment:
Kindly add these required files for your linters setup:
- .eslintrc.json
- .hintrc
- .stylelintrc.json


You can have access to these files [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js) or in your general requirements of your project you can have access to these files.